subprojects/core/src/main/java/org/gradle/invocation/DefaultGradle.java

```diff
@@ -312,7 +313,7 @@ public void afterProject(Action<? super Project> action) {
     }
 
     @Override
-    public void beforeSettings(Closure<?> closure) {
+    public void beforeSettings(@DelegatesTo(Settings.class) Closure<?> closure) {
```

Comment:
This is wrong. The method in the interface isn't annotated, since the rule detected correctly that the Javadoc states that is passes settings as a parameter. Though the actual implementation doesn't have any Javadoc. Can you check the Javadoc from the interface here?

Reply:
Removed from `DefaultGradle`